src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs

```diff
@@ -35,7 +37,9 @@ public abstract class AzureServiceBusMessagePump<TMessage> : MessagePump<TMessag
             : base(configuration, serviceProvider, logger)
         {
             Settings = serviceProvider.GetRequiredService<AzureServiceBusMessagePumpSettings>();
+            JobId = Settings.Options.JobId;
 
+            _subscriptionName = $"{Settings.SubscriptionPrefix}:{JobId}";
```

Comment:
Let's use a hyphen instead, this is more commonly used
```suggestion
            _subscriptionName = $"{Settings.SubscriptionPrefix}-{JobId}";
```